### ISSUE TITLE
Getting this error: "Fatal error: failed to mount FUSE fs: fusermount: exec: "fusermount": executable file not found in $PATH"

### ISSUE THREAD

**Issue by captainfuckface** [Magisk-Modules-Repo/com.piyushgarg.rclone#62]
Sorry if this is a dumb question I'm kind of a noob but I followed along with the instructions posted and when I try to use the mount command I always get that error.  Using LineageOS on a Galaxy Tab S

**Comment by mirsella** [comment]
are you on android 11 ?

**Comment by mirsella** [comment]
i fixed it by putting a fusermount [binary](https://forum.xda-developers.com/t/fusermount-on-android-rclone-mount.3866652/) in /system/bin.

are you on android 11 ?

**Comment by mirsella** [comment]
you have to manually put the binary in /system/bin from TWRP. 

download the fusermount binary from this repo or from [here](https://forum.xda-developers.com/t/fusermount-on-android-rclone-mount.3866652/)
put it in your phone storage.
reboot to twrp, mount system.
go into the terminal and type : `cp /sdcard/fusermount /system/bin`
and `chmod +x /system/bin/fusermount` to make it executable.

don't hesitate to ask me a question.

**Comment by everything13** [comment]
> you have to manually put the binary in /system/bin from TWRP.
> 
> download the fusermount binary from this repo or from [here](https://forum.xda-developers.com/t/fusermount-on-android-rclone-mount.3866652/)
> put it in your phone storage.
> reboot to twrp, mount system.
> go into the terminal and type : `cp /sdcard/fusermount /system/bin`
> and `chmod +x /system/bin/fusermount` to make it executable.
> 
> don't hesitate to ask me a question.

Can this be done on non-root android on the termux app?

**Comment by mirsella** [comment]
> Can this be done on non-root android on the termux app?
yes without root but you'll need a custom recovery (like TWRP). termux can't do anything about it if he doesn't have root acess.
you made me curious why using (or just commenting) on this magisk module if you don't use magisk ?
